linters/stylua/test_data/stylua_basic.fmt.shot

```diff
@@ -0,0 +1,21 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Why were these added for stylua? Seems like something is broken with versioning...